[----------]
[ RUN      ] ExceptionTest.referenceError
exception_test.js:23
ReferenceError: fooBar is not defined

[  FAILED  ] ExceptionTest.referenceError (1 ms)
[ RUN      ] ExceptionTest.notAFunctionError
exception_test.js:28
TypeError: object is not a function

[  FAILED  ] ExceptionTest.notAFunctionError (1 ms)
[ RUN      ] ExceptionTest.errorInMatcherFactory
exception_test.js:32
TypeError: isNearNumber requires two number arguments

[  FAILED  ] ExceptionTest.errorInMatcherFactory (1 ms)
[ RUN      ] ExceptionTest.stackOverflow
RangeError: Maximum call stack size exceeded

[  FAILED  ] ExceptionTest.stackOverflow (1 ms)
[ RUN      ] ExceptionTest.unknownPropertyOnLongFunction
exception_test.js:45
TypeError: Object function foo() {
    return 'asd';
  } has no method 'bar'

[  FAILED  ] ExceptionTest.unknownPropertyOnLongFunction (1 ms)
[ RUN      ] ExceptionTest.objectLiteralException
{ name: 'SomeException' }

[  FAILED  ] ExceptionTest.objectLiteralException (1 ms)
[ RUN      ] ExceptionTest.customExceptionClassWithoutStack
{}

[  FAILED  ] ExceptionTest.customExceptionClassWithoutStack (1 ms)
[ RUN      ] ExceptionTest.customExceptionClassWithStack
exception_test.js:63
{}

[  FAILED  ] ExceptionTest.customExceptionClassWithStack (1 ms)
[ RUN      ] ExceptionTest.customExceptionClassWithToString
SomeException

[  FAILED  ] ExceptionTest.customExceptionClassWithToString (1 ms)
[ RUN      ] ExceptionTest.passingTest
[       OK ] ExceptionTest.passingTest (1 ms)
[----------]

[----------]
[ RUN      ] ThrowingConstructorTest.someTest
exception_test.js:77
Error: taco

[  FAILED  ] ThrowingConstructorTest.someTest (1 ms)
[----------]

[  FAILED  ]
